spec/features/saml/saml_spec.rb

```diff
@@ -1,9 +1,12 @@
 require 'rails_helper'
 
-feature 'saml api' do
+class MockSession; end
```

Comment:
I think we could probably get away with just one happy path spec that tests that CloudHSM is called instead of running through all the specs with CloudHSM.

Reply:
eventually cloudhsm is on and all the saml tests will be running with it on.